components/organisms/ContributorProfilePage/contributor-profile-page.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
```

Comment:
IMHO remove global eslint disables and at worst, only ignore the line in question